team.go

```diff
@@ -54,6 +54,8 @@ type Team struct {
 	Permissions        *TeamPermissions    `jsonapi:"attr,permissions"`
 	UserCount          int                 `jsonapi:"attr,users-count"`
 	SSOTeamID          string              `jsonapi:"attr,sso-team-id"`
+	// AllowMemberTokenManagement is false for TFE versions older than v202408
```

Comment:
why is this the case?

Reply:
This PR will go out before the release does, so we want to make it clear when this is available!

Reply:
When the API response is deserialized, for versions of TFE which dont know about the new feature `allow-member-token-management` will be absent. So when we initialize this object, `AllowMemberTokenManagement` wont be set, and `bool` s default to false

The comment is worthwhile because this is misleading! Checks for `team. AllowMemberTokenManagement == true` will fail, even though team members _can_ manage tokens.

We need to help users understand that this field is not useful pre TFE v202408

Reply:
gotchyyaaa, i was confused because before this release all members should be able to create tokens and is true by default. but this makes sense for TFE though, i was just tripping myself up